README.md

```diff
@@ -2,62 +2,65 @@
 
```

Comment:
@tanberry, can you review the contents of `.md` files in this Pull Request? Thanks in advance!